frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -8,26 +8,27 @@
 
 @RestController
 public class CarController {
-
-    private List<Car> currentCarList = new ArrayList<>();
     Map<Integer, Car> map = new HashMap<>();
 
     @PostMapping(path = "/car", consumes = {"application/json"})
-    public List<Car> addCar(@RequestBody Car car) {
-        currentCarList.add(car);
-        return currentCarList;
+    public Map<Integer,Car> addCar(@RequestBody Car car) {
+        map.put(map.size()+1,car);
```

Comment:
This isn't compatible with the update single car method. E.g. I can set some other index as the car id in the other one and add it with any key I like. Not with the order index.